lib/StripeResource.js

```diff
@@ -112,43 +164,8 @@ StripeResource.prototype = {
       });
       res.once('end', () => {
         const headers = res.headers || {};
-        // NOTE: Stripe responds with lowercase header names/keys.
```

Comment:
Should we keep this note in _addHeadersDirectlyToResponse?

Reply:
Sure!